libs/thermometer/docs/reference/input/on-temperature-condition-changed.md

```diff
@@ -1,15 +1,15 @@
-# on Temperature Condition Changed
+# on Temperate Condition Changed
```

Comment:
That's what you called it:

void onTemperateConditionChanged(TemperatureCondition condition, int temperature, TemperatureUnit unit, Action handler)